tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/GoLanguageService.Checks.cs

```diff
@@ -0,0 +1,156 @@
+using Azure.Sdk.Tools.Cli.Helpers;
+using Azure.Sdk.Tools.Cli.Models.Responses.Package;
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Services.Languages;
+
+namespace Azure.Sdk.Tools.Cli.Services.Languages;
+
+/// <summary>
+/// Go-specific implementation of language repository service.
+/// Uses tools like go build, go test, go mod, gofmt, etc. for Go development workflows.
+/// </summary>
+public partial class GoLanguageService : LanguageService
```

Comment:
Is the file name correct?

Reply:
Yes the common convention for partial classes that you want to split out into another file you use the class name and then add the part of the class implemented like `.Checks`